new/LIT7121PrayerChildbirth.xml

```diff
@@ -0,0 +1,80 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7121PrayerChildbirth" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ጸሎት፡ በእንተ፡ ሕማመ፡ ወሊድ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Ṣalot baʾənta ḥəmāma walid</title>
+                <title xml:lang="en" corresp="#t1">Prayer for the pain of birthgiving</title>
```

Comment:
We used to translate baʾənta with "against", if it something negative, even if the preposition "for" is idiomatically perfectly OK in this context.

Reply:
Maybe "against" is even more idiomatic here, though! Thanks, I will change it :)